docs/man/man8/unitd.8.in

```diff
@@ -1,10 +1,16 @@
-.\" (C) 2017-2023, NGINX, Inc.
-.\" (C) 2017-2023 Andrei Zeliankou
-.\" (C) 2018-2023 Konstantin Pavlov
-.\" (C) 2021-2023 Zhidao Hong
+.\" (C) 2017-2024, NGINX, Inc.
```

Comment:
super nitpick, feel free to ignore, not a blocker, why does this line have a , after the years, but not the lines with the individuals' names? 🙂

Reply:
Occurs again below...

Reply:
LGTM minus fixing the spurious initial commas in "2024, NGINX, Inc."

Reply:
No idea, will remove this one.